### DISCUSSION TITLE
Linux Notification

### DISCUSSION THREAD

**Discussion by rakshit-puri** [skate1512/Toggle_Keys_Notification#8] [General]
We can use this for any doubts and questions.

**Comment by kedar-hargude** [comment]
Okay so I have managed to write a script which notifies whether the caps and num lock are on or not. 
I use Fedora, so have created everything that way. Although this should work for every flavour of Linux, I have not tested it due to lack of access.

**Reply by rakshit-puri** [reply]
I'll try to test it with some other Linux OS.

**Comment by kedar-hargude** [comment]
I also have to include a video recording. This is my first time contributing, had thought that the pull request that I made would link in my forked repo. 
Also, running in the background in Linux is way more complicated than that of Windows. I have read that it is different for different versions like Ubunto, Fedora, Kali and all.

That said, the given script works completely fine.

**Comment by kedar-hargude** [comment]
I wanted to ask you @skate1512, the issue #5 is in the issue tab, and when I created a pull request, it created a new issue #7. How can I take care from the next time so that I'll be providing to only the specific issue when I am creating a pull request? I mean, when my pull request gets accepted, it should show in the issue #5 right?

**Reply by rakshit-puri** [reply]
The issue is #5, and the Pull request is #7. You can mention the issue number while creating a PR.

**Reply by kedar-hargude** [reply]
by just adding #5 in the title??

**Reply by rakshit-puri** [reply]
Add it in the Title and in the Description with a short description. You can also add screenshots.

**Comment by rakshit-puri** [comment]
@kedar-hargude  Did your pull request count increase for Hacktoberfest?

**Reply by kedar-hargude** [reply]
Before your project, I had created only 1 PR and it got accepted. And only that showed in the profile. I didn't get your question completely though. Like, what specifically did you want to ask.

**Reply by rakshit-puri** [reply]
I added the label hacktoberfest-accepted for your PR so it should increase your total PR. Check if this project is coming in your PR list on the hacktoberfest website.

**Reply by rakshit-puri** [reply]
According to the site, "PRs not showing up? Hacktoberfest profiles refresh approximately every 15 minutes if you’re reloading this page, or every 6 hours if not."

**Reply by kedar-hargude** [reply]
Thanks for adding the label. Although, from what I have seen is, once you accept the pull request, only then it gets added to your profile. Earlier, before 2020, only PR count was used. Since 2020, the PR accepted count is used.

**Reply by rakshit-puri** [reply]
Ok, I will test the script soon and then accept it.
